test/ocsp/helper/helper.go

```diff
@@ -0,0 +1,187 @@
+package helper
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/base64"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"golang.org/x/crypto/ocsp"
+)
+
+var method = flag.String("method", "GET", "Method to use for fetching OCSP")
+var urlOverride = flag.String("url", "", "URL of OCSP responder to override")
+var tooSoon = flag.Int("too-soon", 76, "If NextUpdate is fewer than this many hours in future, warn.")
+var ignoreExpiredCerts = flag.Bool("ignore-expired-certs", false, "If a cert is expired, don't bother requesting OCSP.")
+var expectStatus = flag.Int("expect-status", 0, "Expect response to have this numeric status (0=good, 1=revoked)")
+
+func getIssuer(cert *x509.Certificate) (*x509.Certificate, error) {
+	if len(cert.IssuingCertificateURL) == 0 {
+		return nil, fmt.Errorf("No AIA information available, can't get issuer")
+	}
+	issuerURL := cert.IssuingCertificateURL[0]
+	resp, err := http.Get(issuerURL)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
+	}
+	var issuer *x509.Certificate
+	if strings.Join(resp.Header["Content-Type"], "") == "application/x-pkcs7-mime" {
```

Comment:
This works but seems a bit hacky, any reason not to just fail if we passed multiple `Content-Type` headers?